### ISSUE TITLE
Bug: NamespaceDeclaration sniff false positive with namespace operator

### ISSUE THREAD

**Issue by jrfnl** [slevomat/coding-standard#543]
The `namespace` keyword has two uses:
* To declare a namespace.
* To explicitly request something from the current namespace.
See: http://php.net/manual/en/language.namespaces.nsconstants.php

This second use case is not taken into account by the `SlevomatCodingStandard.Namespaces` sniff leading to false positives.

Example code:
```php
<?php
namespace MyProject;

function something() { 
    // code
}

$output = namespace\something();
```

Will throw the following false positives:
```
- Only one namespace in a file is allowed.
  (SlevomatCodingStandard.Namespaces.RequireOneNamespaceInFile.MoreNamespacesInFile)
- Expected one space after namespace statement.
  (SlevomatCodingStandard.Namespaces.NamespaceDeclaration.InvalidWhitespaceAfterNamespace)
- Disallowed content between namespace name and semicolon.
  (SlevomatCodingStandard.Namespaces.NamespaceDeclaration.DisallowedContentBetweenNamespaceNameAndSemicolon)
```

Same problem existed upstream in at least one sniff: https://github.com/squizlabs/PHP_CodeSniffer/issues/2194

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/0addd87c006c9ed9b9a270fabf11f18a384a1f0a and https://github.com/slevomat/coding-standard/commit/ae4cb266c78b4ab95fd4b6254748f279d54e2247

**Comment by jrfnl** [comment]
:+1: Thanks!
